charts/access/mattermost/tests/configmap_test.yaml

```diff
@@ -0,0 +1,18 @@
+suite: Test deployment
+templates:
+  - configmap.yaml
+tests:
+  - it: should match the snapshot
+    set:
+      teleport:
+        address: teleport.example.com:1234
+      mattermost:
+        url: https://my.mattermost.com
+        token: test-mattermost-token
+        recipients:
+          - security@mycompany.com
```

Comment:
Best to stick to example.com from https://tools.ietf.org/search/rfc2606#section-3.  mycompany.com is a real domain who probably doesn't want this mail.